presto-hive/src/main/java/io/prestosql/plugin/hive/authentication/NoHiveMetastoreAuthentication.java

```diff
@@ -15,11 +15,13 @@
 
 import org.apache.thrift.transport.TTransport;
 
+import java.util.Optional;
+
 public class NoHiveMetastoreAuthentication
         implements HiveMetastoreAuthentication
 {
     @Override
-    public TTransport authenticate(TTransport rawTransport, String hiveMetastoreHost)
+    public TTransport authenticate(TTransport rawTransport, String hiveMetastoreHost, Optional<String> delegationToken)
```

Comment:
`checkArgument(!delegationToken.isPresent(), "delegation token is not supported")`